components/Footer/__tests__/__snapshots__/Footer.test.js.snap

```diff
@@ -196,7 +196,7 @@ exports[`Footer should render with no props passed 1`] = `
       className="row copyright"
     >
       Copyright 
-      2018
+      2019
```

Comment:
Good catch. Wanna just do `{(new Date()).getFullYear()}`?

Reply:
JSX, in a snapshot?

Reply:
Oh sorry - the actual source code.

Reply:
oh wait that's already there and the snapshot updated 😂